client/python/samples/resnet_grpc_predict_numpy.py

```diff
@@ -54,14 +57,13 @@
     iterations = imgs.shape[0]
 
 for i in range (iterations):
-    # preparing predict request
+    # preparing inputs
     inputs = {
         input_name: [imgs[i%imgs.shape[0]]]
     }
-    request = make_grpc_predict_request(inputs, model_name, model_version)
 
     # sending predict request and receiving response
-    response = client.predict(request)
+    response = client.predict(inputs, model_name, model_version, timeout)
```

Comment:
response is now either a numpy array or a dict with outputs
are you sure next steps - resnet_postprocess will work now?